src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/IssueNavigatorPage.kt

```diff
@@ -36,11 +36,11 @@ class IssueNavigatorPage(
     }
 
     fun getIssueKeys(): Set<String> {
-        return driver
-            .findElements(By.className("issue-link-key"))
-            .map { it.text }
```

Comment:
each map invocation here means two remote calls

Reply:
this was found during VU profiling

Reply:
Did it help? What's the before and after?
I recall that the problem at hand is about VUs being the bottleneck after removing wildcard JQLs. Is that accurate?

Reply:
No, that's another problem. With this (and other related) changes, we are reaching 500 000 processed actions in the timespan of 20 minutes. The same tests without these changes were closer to 400 000 actions, all other things being the same.